public/app/themes/justice/inc/mail.php

```diff
@@ -48,7 +49,7 @@ function mail_template_default($templates, $attrs)
     ];
 
     // Don't short-circuit if the password doesn't look right
-    $api_key = env('GOV_NOTIFY_API_KEY');
+    $api_key = Config::get('GOV_NOTIFY_API_KEY');
```

Comment:
Just a quick one... is this because `GOV_NOTIFY_API_KEY` is not available as a PHP constant?

Reply:
... AH! I've just answered my own question!! Haha. 😄